Stagging

rdvargas40 opened the pull request:
Los cambios que solicitó

rdvargas40 committed:
DRY with pdf estimations

Used prob_density_estimate inside
max_post_estimate

rdvargas40 committed:
Ignore pycache

rdvargas40 committed:
Complement Readme

rdvargas40 committed:
Include arviz on requirements

rdvargas40 committed:
Cambios Parra

Cambios Solicitados por Parra

Davidcparrar commented on bayesian_asimetrix/statistics.py:
```diff
@@ -48,11 +48,8 @@ def max_post_estimate(df_post: pd.DataFrame) -> pd.Series:
-    map_theta = dp_post[dp_post.pdf == dp_post.pdf.max()].iloc[0][params]
+    pdfs = prob_density_estimate(df_post)
+    map_theta = df_post.iloc[np.where(pdfs == pdfs.max())]
     map_theta.name = 'MAP'
```
Si map_theta es un dataframe es mejor definirlo como 
```suggestion
    map_theta['name']= 'MAP'
```
Tengo entendido que en algunas versiones de python/pandas esa notación puede fallar

Davidcparrar commented on .gitignore:
```diff
@@ -0,0 +1 @@
+__pycache__
```
Cuándo se crea un repo en github, él da la opción de importar un gitignore template para diferentes lenguajes de programación. Le recomiendo el de python porque eso le ahorra la pensada de ignorar varias cosas. https://github.com/github/gitignore/blob/main/Python.gitignore

Davidcparrar reviewed (approved):
De los cambios que puso no tengo casi comentarios. Son más sugerencias.

rdvargas40 committed:
Tuple Suggestion with Rope

Implemented the Tuple Typing Suggestion David Parra made in the last Code Review

rdvargas40 committed:
Summary Table

A Method to Extract a Summary Table. This is for the analyst to obtain an straightforward diagnose of the posteriori distribution
